### ISSUE TITLE
[TW-1072] allow datetime as due value

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1107]
_Peter De Poorter on 2010-10-23T05:25:23Z says:_ 

 Current situation:
   
     due:<due-date>
          Specifies the due-date of a task.

Goal: to be able to use a datetime format as due value

Example: task 3 due:25/10/2010-10:00:00

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-10-23T05:25:23Z
Modified: 2014-02-09T02:17:34Z
```

**Comment by taskwarrior** [comment]
_Johannes Schlatow on 2010-10-23T08:24:54Z says:_ 

 You have to specify the dateformat in your .taskrc:

<pre>
dateformat=d/m/Y-H:N:S
</pre>

See also the taskrc.5 manpage.

Nevertheless it would be nice to be able to abbreviate due dates so that I can specify the dateformat above and type this:

<pre>
$ task add one due:25/10/2010-10
</pre>

which will expand to 25/10/2010-10:00:00

**Comment by taskwarrior** [comment]
_Peter De Poorter on 2010-10-23T14:46:27Z says:_ 

 in the taskrc manpage I found:

dateformat=m/d/Y

I've tested using the mentioned dateformat in my .taskrc but it doesn't work.
I use task 1.9.2. Is it perhaps a feature of a newer task version ?

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2010-10-23T16:23:54Z says:_ 

 <pre>
$ task add foo due:31/10/2010-14:30:00
Created task 1.
$ task list

ID Project Pri Due                 Active Age     Description                                  
 1             31/10/2010-14:30:00         2 secs foo
</pre>

**Comment by taskwarrior** [comment]
_Peter De Poorter on 2010-10-24T03:14:34Z says:_ 

 I use task 1.9.2. If I do the command I get:

task add foo due:31/10/2010-14:30:00
"31/10/2010-14:30:00" is not a valid date in d/m/Y format.

I've searched in the feature list of 1.9.3 and I've found feature #163 time of day support. This will be the reason
of the difference. So I asked a feature that's already implemented in the new beta version. Good news for me.
Thank you for implementing it !
